.gitlab/ci/on-push.yml

```diff
@@ -70,14 +70,15 @@ run-unittests-and-lint:
     - if: '$BUCKET_UPLOAD == "true"'
       when: never
     - if: '$DEMISTO_SDK_NIGHTLY != "true"'
+    - if: '$CI_PIPELINE_SOURCE =~ /^(push|contrib)$/'
 
 
 # runs in circle for the on-push flow (because we need to run it there for contributors anyways)
 run-validations:
   extends:
     - .run-validations
   rules:
-    - if: '$NIGHTLY'
+    - if: '$CI_PIPELINE_SOURCE =~ /^(push|contrib)$/'
```

Comment:
what is this change?

Reply:
entering in cases of on-push or contribution.